.bazelrc

```diff
@@ -0,0 +1,16 @@
+# For bazel-in-bazel testing
+# Trick bazel into treating BUILD files under examples/* as being regular files
+# This lets us glob() up all the files inside the examples to make them inputs to tests
+# (Note, we cannot use `common --deleted_packages` because the bazel version command doesn't support it)
+# To update these lines, run tools/bazel_integration_test/update_deleted_packages.sh
+build --deleted_packages=examples/simple,examples/simple/mockascript,examples/simple/mockascript/internal,examples/simple/tests
+query --deleted_packages=examples/simple,examples/simple/mockascript,examples/simple/mockascript/internal,examples/simple/tests
+
+# Import Shared settings
+import %workspace%/shared.bazelrc
+
+# Import CI settings.
+import %workspace%/ci.bazelrc
```

Comment:
I don't think this belongs, CI settings are exclusive from what users use

Reply:
The file is loaded for everyone. However, the settings in the file are all suffixed with `:ci`. In other words, they only take effect when the `ci` config is specified. This happens [here](https://github.com/cgrindel/rules_bazel_integration_test/blob/main/.github/workflows/bazel.yml#L19). I use this pattern so that everyone can easily see what configuration values are used for CI vs development.

Reply:
I see, thanks